.github/workflows/rust.yml

```diff
@@ -33,7 +33,7 @@ jobs:
     needs: build
     strategy:
       matrix:
-        version: [ 20.10.5-focal, 21.10.0-focal, ci ]
+        version: [ previous-lts, lts, latest ]
```

Comment:
why `latest` and not `ci`? `latest` is going to be the same as `lts` most of the time

Reply:
It seems the `ci` tag is no longer updated unlike `latest`

Reply:
CI is only build from master, but something is wrong with the tags...looking into it, so think we should stick to `ci`.

Reply:
Done